nifi-manifest/nifi-runtime-manifest-test/src/test/java/org/apache/nifi/runtime/manifest/TestRuntimeManifest.java

```diff
@@ -264,6 +265,7 @@ void testRuntimeManifest() throws IOException {
         assertNotNull(routeOnAttributeDef.getDynamicRelationship().getDescription());
 
         assertTrue(routeOnAttributeDef.getSupportsDynamicProperties());
+        assertTrue(routeOnAttributeDef.getSupportsSensitiveDynamicProperties());
```

Comment:
Seems like this one is causing a test failure because RouteOnAttribute doesn't support sensitive dynamic properties. Probably have to locate a different definition for a component we know supports it, since all the ones currently in the test don't (I think).

```
[ERROR] Failures:
[ERROR]   TestRuntimeManifest.testRuntimeManifest:268 expected: <true> but was: <false>
```

Reply:
> Seems like this one is causing a test failure because RouteOnAttribute doesn't support sensitive dynamic properties. Probably have to locate a different definition for a component we know supports it, since all the ones currently in the test don't (I think).
> 
> ```
> [ERROR] Failures:
> [ERROR]   TestRuntimeManifest.testRuntimeManifest:268 expected: <true> but was: <false>
> ```

Yep, my bad.  Looks like ExecuteSQL has the annotation; added an assertion for that one.

Reply:
Ah nice, it looks good locally, will wait for CI builds